data/tests/test_teleicare_history_importer.py

```diff
@@ -210,6 +213,13 @@ def test_create_declarations_from_history(self, mocked_add_composition_function,
 
         version_declaration_to_create_as_declaration.refresh_from_db()
         created_declaration = Declaration.objects.get(siccrf_id=CA_to_create_as_declaration.cplalim_ident)
+        self.assertEqual(
+            created_declaration.company,
+            EtablissementToCompanyRelation.objects.get(
+                siccrf_id=etablissement_to_create_as_company.etab_ident
+            ).company,
+        )
+        # self.assertEqual(created_declaration.mandated_company, None)
```

Comment:
Pourquoi c'est commenté ? Si je comprends bien le test ça pourrait être pertinent non ? on pourrait même utiliser `assertIsNone`
```suggestion
        self.assertIsNone(created_declaration.mandated_company)
```